ViewModel/HyvaGridViewModel.php

```diff
@@ -89,9 +94,19 @@ private function buildColumnDefinitions(): array
         $keysToHide       = $this->getGridDefinition()->getExcludedColumnKeys();
         $availableColumns = $this->getGridSourceModel()->extractColumnDefinitions($columns, $keysToHide, $showAll);
 
+        $availableColumns = $this->preprocessColumnDefinitions($availableColumns);
+
         return zip($this->getColumnKeys($availableColumns), values($availableColumns));
     }
 
+    private function preprocessColumnDefinitions($columnDefinitions): array
```

Comment:
Please add an `array` type to the `$columnDefinition` argument in the signature.